hasura.planx.uk/migrations/1707234695689_create_view_feedback_summary/up.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE VIEW "public"."feedback_summary" AS
+SELECT 
+    fb.id AS feedback_id,
+    t.name AS team,
+    f.slug AS service_slug,
+    fb.created_at,
+    fb.node_id,
+    fb.device,
+    fb.user_context,
+    fb.user_comment,
+    fb.feedback_type,
+    fb.status,
+    fb.node_type,
+    COALESCE(
+        published_flow_node.data ->> 'title', 
+        published_flow_node.data ->> 'text', 
+        published_flow_node.data ->> 'flagSet'
+    ) AS node_title,
+    published_flow_node.data ->> 'description' AS node_text,
+    published_flow_node.data ->> 'info' AS help_text,
+    published_flow_node.data ->> 'policyRef' AS help_sources,
+    published_flow_node.data ->> 'howMeasured' AS help_definition,
+    (fb.user_data -> 'passport' -> 'data' -> '_address' ->> 'single_line_address') AS street_address,
```

Comment:
nit: plain `address` is more consistent with how we call this other places. 

Perhaps also another good use case of `COALESCE` because this will differ depending if a user selects an OS address or plots their own new address:
```
COALESCE(
  fb.user_data -> 'passport' -> 'data' -> '_address' ->> 'single_line_address',
  fb.user_data -> 'passport' -> 'data' -> '_address' ->> 'title'
) as address,
```